react-components/src/hooks/useClickedNode.tsx

```diff
@@ -82,25 +85,11 @@ export const useClickedNodeData = (): ClickedNodeData | undefined => {
     };
   }, [viewer]);
 
-  const nodeDataPromises = useFdm3dNodeDataPromises(
-    intersection?.model.modelId,
-    intersection?.model.revisionId,
-    intersection?.type === 'cad' ? intersection.treeIndex : undefined
-  ).data;
-
-  const assetMappingResult = useAssetMappingForTreeIndex(
-    intersection?.model.modelId,
-    intersection?.model.revisionId,
-    intersection?.type === 'cad' ? intersection.treeIndex : undefined
-  ).data;
-
-  const pointCloudAssetMappingResult = usePointCloudAnnotationMappingForAssetId(
-    intersection?.model.modelId,
-    intersection?.model.revisionId,
-    intersection?.type === 'pointcloud'
-      ? intersection.assetRef?.externalId ?? intersection.assetRef?.id
-      : undefined
-  ).data;
+  const nodeDataPromises = useFdm3dNodeDataPromises(intersection).data;
```

Comment:
It a bit more clean to have it as below? for other two Mapping as well
`const { data: nodeDataPromises } = useFdm3dNodeDataPromises(intersection);`